mlrun/datastore/utils.py

```diff
@@ -180,3 +181,53 @@ def get_kafka_brokers_from_dict(options: dict, pop=False) -> typing.Optional[str
             FutureWarning,
         )
     return kafka_bootstrap_servers
+
+
+def transform_list_filters_to_tuple(additional_filters):
+    tuple_filters = []
+    if not additional_filters:
+        return tuple_filters
+    validate_additional_filters(additional_filters=additional_filters)
```

Comment:
Minor style thing, but I think this is unnecessary boilerplate.
```suggestion
    validate_additional_filters(additional_filters)
```